test/cpp/jit/test_gpu.cpp

```diff
@@ -1,4 +1,4 @@
-#if defined(USE_CUDA)
+// #if defined(USE_CUDA)
```

Comment:
Just curious. why did you disable this?

Reply:
Left over from working on it. VSCode won't link functions/classes when surrounded by the if def.